webpack.config.js

```diff
@@ -65,17 +61,20 @@ module.exports = {
     modules: [path.resolve(__dirname, 'src'), 'node_modules']
   },
   externals: {
-    'playkit-js': {
-      commonjs: 'playkit-js',
-      commonjs2: 'playkit-js',
+    '@playkit-js/playkit-js': {
+      commonjs: '@playkit-js/playkit-js',
+      commonjs2: '@playkit-js/playkit-js',
       amd: 'playkit-js',
-      root: ['playkit', 'core']
+      root: ['KalturaPlayer', 'core']
     },
     'playkit-js-providers': {
       commonjs: 'playkit-js-providers',
       commonjs2: 'playkit-js-providers',
       amd: 'playkit-js-providers',
-      root: ['playkit', 'providers']
+      root: ['KalturaPlayer', 'providers']
     }
+  },
+  optimization: {
+    minimize: PROD
```

Comment:
use `--mode production`/`--mode development` in package.json instead(for build/dev/watch scripts)

Reply:
done